src/types/index.ts

```diff
@@ -32,7 +32,7 @@ export interface TemplateLintMessage {
 export type LintResult = ESLint.LintResult | TemplateLintResult;
 export type LintMessage = Linter.LintMessage | TemplateLintMessage;
 
-export type FilePath = string;
```

Comment:
In theory we could bring this back as an export, just to prevent it from being breaking 🤔

Reply:
Good call. I'll do that.

Reply:
Done